index.htm

```diff
@@ -378,42 +389,80 @@ <h4 class="modal-title">Traffic Count Types</h4>
+                    <div class="modal-header">
+                        <button aria-hidden="true" class="close" data-dismiss="modal" type="button">&times;</button>
+                        <h4 class="modal-title">Record Download Help Guide</h4>
+                    </div>
+                    <div class="modal-body">
+                    <div class="clearfix">
+                        <p>Please <i class="glyphicon glyphicon-check"></i>&nbsp;select any traffic counts that you wish to download. The counts you select will be downloaded as a zipped directory (.zip) with a PDF report for each of the requested counts as well as a comma-separated file (.csv) containing the summary data for all the counts. Click the <button class="btn btn-primary btn-xs active">download selected</button> once you have selected all your records.<br><br>
+                         All <i class="glyphicon glyphicon-unchecked"></i>&nbsp;unchecked records will not be included in the download file.
+                        <img src="lib/images/download.png"> 
+                        </p>
+                    </div>
+                    </div><!-- /.modal-body -->
+                </div><!-- /.modal-content -->
+            </div><!-- /.modal-dialog -->
+        </div><!-- /.modal fade-->
+
        <!-- AGS link Modal -->
         <div class="modal fade" id="ags-modal">
             <div class="modal-dialog" style="width:710px;">
                 <div class="modal-content">
                     <div class="modal-header">
                         <button aria-hidden="true" class="close" data-dismiss="modal" type="button">&times;</button>
-                        <h4 class="modal-title">ArcGIS Server Map Services</h4>
+                        <h4 class="modal-title">ArcGIS Server map services</h4>
                     </div>
                     <div class="modal-body">
-                        This area is under construction... <img src="http://www.dvrpc.org/webmaps/img/dataportal/Transportation_service.png"> 
-                        <!--      <a href="http://arcgis.dvrpc.org/arcgis/rest/services/Transportation/TrafficCounts/MapServer" -->
-                        <a href="http://dvrpc.dvrpcgis.opendata.arcgis.com/datasets/8eb412d170af43e58dbb9ff382e71777_0?uiTab=table&orderByFields=CO_NAME+ASC&where=CO_NAME+like+%27%25Chester&geometry=-77.42%2C39.687%2C-73.822%2C40.319" target="_new"><button class="btn btn-primary btn-lg">Access Map Service</button></a>
+                     <div class="clearfix">
+                        
+                        <div>
+                        <img src="lib/images/ags_connect.png" style="float:right;"> 
+                        <b>ArcGIS for Desktop:</b> Copy and paste the URL below to access our ArcGIS Server map services.
+                        <pre><code>http://arcgis.dvrpc.org/arcgis/rest/services</code></pre>
+                        </div>
+                        <br><br><br><br><br><br><br>
```

Comment:
Dude. Really? What is this? Use proper syntax, code formatting, and your existing styles to build things or go old school and use a table.